doc/source/guide/default.rst

```diff
@@ -0,0 +1,52 @@
+********************************
+How To Use Config Class 
+********************************
+
+In this tutorial, we look at the functionality of the global Config class.
+
+.. code-block:: python
+
+    df = pd.read_csv("https://github.com/lux-org/lux-datasets/blob/master/data/hpi.csv")
+    df
+
+The dataframe initially registers a few default recommendations, such as Correlation, Enhance, Filter, etc, and defaults to a pandas display.
+
+Registering Custom Actions
```

Comment:
This is probably beyond the scope of the PR, but what do you think about making default_display a boolean so that only pandas and lux could be selected (0 and 1 respectively). This removes user error from misspelling "lux" or "pandas" (albeit the possibility of that happening is minuscule). You can maybe make the variable more descriptive -- something like `default_display_to_lux`. Thoughts? @caitlynachen @dorisjlee

Reply:
That makes sense. This is possibly a good idea.